packages/token-generation-readmodel-checker/src/utils/utils.ts

```diff
@@ -0,0 +1,1218 @@
+/* eslint-disable no-console */
```

Comment:
Temporary for debugging

Reply:
That was to see what gets printed in the tests with `console.log`. I'm gonna remove all the `console.log` before merging.